chore(ruby): Update outdated documentation (#15599)
newrelic/docs-website: src/content/docs/apm/agents/ruby-agent/attributes/enable-disable-attributes-ruby.mdx

* The `url` attribute of [slow SQL traces](/docs/apm/applications-menu/monitoring/view-slow-query-details)
* The `uri` attribute of [external request segments](/docs/agents/ruby-agent/features/http-client-tracing-ruby)

> New Relic recommends having these URIs reported, as they can contain useful debugging information. If your URIs contain sensitive data that you don't want reported, you can disable URI reporting. Sensitive data could include email addresses or account IDs. To disable URI reporting, add any of the above attribute names to the `attributes.exclude` list.

Reviewer: ```suggestion
New Relic recommends having these URIs reported, as they can contain useful debugging information. If your URIs contain sensitive data you don't want to report, such as email addresses or account IDs, you can disable URI reporting. To disable URI reporting, add any of the above attribute names to the `attributes.exclude` list.
```
